public/app/features/logs/components/LogRows.tsx

```diff
@@ -163,7 +158,7 @@ class UnThemedLogRows extends PureComponent<Props, State> {
                 getLogRowContextUi={getLogRowContextUi}
                 row={row}
                 showContextToggle={showContextToggle}
-                showRowMenu={!contextIsOpen}
+                showRowMenu={true}
```

Comment:
This is always true now, so I think we can remove `showRowMenu` prop as it is not used anymore to hide menu for selected log line when context is open.

Reply:
Good 👀 ! Removed in a03cabc